daemon/network.go

```diff
@@ -1044,12 +1044,12 @@ func getEndpointPortMapInfo(ep *libnetwork.Endpoint) (nat.PortMap, error) {
 }
 
 // buildEndpointInfo sets endpoint-related fields on container.NetworkSettings based on the provided network and endpoint.
-func buildEndpointInfo(networkSettings *internalnetwork.Settings, n *libnetwork.Network, ep *libnetwork.Endpoint) error {
+func buildEndpointInfo(settings *networkSettings.Settings, n *libnetwork.Network, ep *libnetwork.Endpoint) error {
```

Comment:
That's where I'm not a big fan of this naming change. This function is called from `container_operations.go`, where `networkSettings` is already used, and that name is derived from container's `NetworkSettings` field.